general-configuration-guide/use-tinymce-distraction-free.md

```diff
@@ -6,7 +6,9 @@ description: Theme that renders a lightweight UI for inline editing.
 keywords: theme inlite distraction-free
 ---
 
-The **Distraction-free** theme renders the editor in lightweight inline mode. This mode provides options to quickly insert links, images, and tables into the content. For a sample view of this mode, see the [Distraction-free editor]({{ site.baseurl }}/demo/editor-dfree/) example page.
+The **Distraction-free** theme renders the editor in lightweight inline mode. This mode provides options to quickly insert links, images, and tables into the content.
```

Comment:
Should we avoid calling distraction-free a "theme" and just go with mode?

Reply:
fixed